docs/source/about/contributing.rst

```diff
@@ -0,0 +1,167 @@
+Contributing
+============
+
+Thank you for taking the time to contribute to this project!
+
+To get started, make sure that you have :code:`pipenv`, :code:`docker` and
+:code:`docker-compose` installed on your computer. Additionally, since this
+is a package that supports both Python 2 and Python 3, please make sure
+you have Python 2.7.15+ and Python 3.6+ installed locally. This project
+assumes that they are available on the system as :code:`python2` and
+:code:`python3`, respectively. If you do not already have them installed,
+consider doing so using `pyenv <https://github.com/pyenv/pyenv>`__.
+
+Integration tests are run against multiple SQL databases, some of which
+require dialect specific installations to be available in the
+development environment. Currently this affects MySQL. A compatible
+driver can be installed on OSX with:
+
+.. code:: bash
+
+   brew install mysql
+
+or on Ubuntu with:
+
+.. code:: bash
+
+   apt-get install python-mysqldb
+
+For more details on other systems please refer to `MySQL dialect
+information <https://docs.sqlalchemy.org/en/latest/dialects/mysql.html>`__.
+
+Once the dev environment is prepared, from the root of the repository,
+run:
+
+::
+
+   docker-compose up -d
+   pipenv sync --dev
+   pipenv shell
+
+   py.test graphql_compiler/tests
+
+Some snapshot and integration tests take longer to setup, run, and
+teardown. These can be optionally skipped during development by running
+the tests with the :code:`--skip-slow` flag:
+
+.. code:: bash
+
+   py.test graphql_compiler/tests --skip-slow
+
+If you run into any issues, please consult the TROUBLESHOOTING.md file.
```

Comment:
`.md` file? Also should this be a link to another page?

Reply:
Since our `TROUBLESHOOTING.md` file includes instructions for troubleshooting issues when setting your contribution development environment, I decided to make it a subsection of our `contributing.rst` file in readthedocs. I will add a link to the troubleshooting section here.